AuctionSite.Bl/User/Validators/UserExistenceValidator.cs

```diff
@@ -0,0 +1,37 @@
+﻿using AuctionSite.BL.Common;
+using AuctionSite.DataAccess;
+using AuctionSite.Shared;
+using AuctionSite.Shared.BindingModel;
+using AuctionSite.Shared.Enums;
+using AuctionSite.Shared.Exceptions;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AuctionSite.BL.User.Validators
+{
+    public class UserExistenceValidator : IValidator<RegisterUserBindingModel>
```

Comment:
ExistenceValidator powinien sprawdzać czy istnieje user, rzucać exception gdy nie istnieje, do tej akcji powinien być moim zdaniem jakiś UserUniquenessValidator, a potem metoda ValidateUniquenessByUsername i ValidateUniquenessByEmail

Reply:
A to już zależy. Możemy u nas przyjąć taką konwencję